src/middleware/dbConnection/test/dbConnection_test.js

```diff
@@ -0,0 +1,82 @@
+import assert from 'assert';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+import mockMbaasClient from './mocks/fhMbaasClientMock';
+
+const mbaasClientStub = sinon.stub();
+const fhDbStub = sinon.stub();
+const middleware = proxyquire('../', {
+  'fh-mbaas-client': {
+    MbaasClient: mockMbaasClient(mbaasClientStub)
+  },
+  'fh-db': {
+    createMongoCompatApi: fhDbStub
+  }
+});
+
+
+describe('Database Connection Middleware', () => {
```

Comment:
since our existing tests are export based, I think it's better to keep them consistent. Can you convert those to use the export format as well, rather than using the bdd format.